### DISCUSSION TITLE
Is the load_var step still experimental?

### DISCUSSION THREAD

**Discussion by tremblaysimon** [concourse/concourse#7391] [General]
Is the load_var step still experimental since RFC about it seems to be merged? In the Concourse documentation https://concourse-ci.org/jobs.html#schema.step.load-var-step.load_var there is a framed text about that. The RFC referenced in the framed texte has been merge for more than one year now: https://github.com/concourse/rfcs/pull/27. 

If I understand the framed texte correctly, since the RFC has been merged, that feature shouldn't be mark as experimental in the documentation or it still is?

**Comment by aoldershaw** [comment]
Thanks for pointing this out! Opened https://github.com/concourse/docs/pull/453 to remove that warning in the docs, and https://github.com/concourse/concourse/pull/7396 to remove the warning in the build logs
